calyx-opt/src/analysis/reaching_defns.rs

```diff
@@ -284,6 +284,150 @@ fn remove_entries_defined_by(set: &mut KilledSet, defs: &DefSet) {
         .collect();
 }
 
+fn build_reaching_def_static(
+    sc: &ir::StaticControl,
+    reach: DefSet,
+    killed: KilledSet,
+    rd: &mut ReachingDefinitionAnalysis,
+    counter: &mut u64,
+) -> (DefSet, KilledSet) {
+    match sc {
+        ir::StaticControl::Empty(_) => (reach, killed),
+        ir::StaticControl::Enable(sen) => {
+            // (Note Caleb/Pai): This is similar to case for enable group right now
```

Comment:
Use the `XXX(username): comment` or `TODO(username): comment` syntax so it's easier to grep for these notes